diff --git a/dist/changesets-assemble-release-plan.cjs.js b/dist/changesets-assemble-release-plan.cjs.js
index e32a5e5d39c3bd920201b5694632d2b44c92d486..a7cf97e5aa5b39ed73e8a6c352fc66fb39fff4c2 100644
--- a/dist/changesets-assemble-release-plan.cjs.js
+++ b/dist/changesets-assemble-release-plan.cjs.js
@@ -322,7 +322,11 @@ function getDependencyVersionRanges(dependentPkgJSON, dependencyRelease) {
         versionRange: // intentionally keep other workspace ranges untouched
         // this has to be fixed but this should only be done when adding appropriate tests
         versionRange === "workspace:*" ? // workspace:* actually means the current exact version, and not a wildcard similar to a reguler * range
-        dependencyRelease.oldVersion : versionRange.replace(/^workspace:/, "")
+        dependencyRelease.oldVersion :
+				versionRange === "workspace:^" ? // workspace:^ means at least as new as the current version but less than then next major
+				`^${dependencyRelease.oldVersion}`
+				:
+				versionRange.replace(/^workspace:/, "")
       });
     } else {
       dependencyVersionRanges.push({
