diff --git a/lib/stream.js b/lib/stream.js
index 67b1b3c81cd764a9dc8b5e0bbb9a782cd32943bb..aed758ae56a5ce6134d46ec2972076e493cfc619 100644
--- a/lib/stream.js
+++ b/lib/stream.js
@@ -3,14 +3,14 @@
  * @returns {Duplex}
  */
 module.exports.getStream = function getStream(ssl) {
-  const net = require('net')
-  if (typeof net.Socket === 'function') {
-    return new net.Socket()
-  } else {
-    const { CloudflareSocket } = require('pg-cloudflare')
-    return new CloudflareSocket(ssl)
+  try {
+    const { CloudflareSocket } = require("pg-cloudflare");
+    return new CloudflareSocket(ssl);
+  } catch {
+    const net = require("net");
+    return new net.Socket();
   }
-}
+};
 
 /**
  * Get a TLS secured socket, compatible with the current environment,
@@ -18,11 +18,11 @@ module.exports.getStream = function getStream(ssl) {
  * @returns {Duplex}
  */
 module.exports.getSecureStream = function getSecureStream(options) {
-  var tls = require('tls')
-  if (tls.connect) {
-    return tls.connect(options)
+  if (options.socket.startTls) {
+    options.socket.startTls(options);
+    return options.socket;
   } else {
-    options.socket.startTls(options)
-    return options.socket
+    var tls = require("tls");
+    return tls.connect(options);
   }
-}
+};
