diff --git a/dist/transports/types.d.ts b/dist/transports/types.d.ts
index 1503db4d835325eaf08baf91d5d88e263b0af04a..3669365574a4e776c8f710afa986c39d57dc5fb1 100644
--- a/dist/transports/types.d.ts
+++ b/dist/transports/types.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="@cloudflare/workers-types" />
 import type { BaseTransportOptions } from '@sentry/types';
 import type { Context } from '../types';
 export type FetchTransportOptions = BaseTransportOptions & {
diff --git a/dist/types.d.ts b/dist/types.d.ts
index d7bd18b5f77dbddd438ee2e957e9724eea5bd37b..061aa19de9f241ed684a7775747c2d560783d0c5 100644
--- a/dist/types.d.ts
+++ b/dist/types.d.ts
@@ -1,4 +1,3 @@
-/// <reference types="@cloudflare/workers-types" />
 import type { ClientOptions, Options as CoreOptions } from '@sentry/types';
 import type { RequestDataOptions } from './integrations';
 import type { FetchTransportOptions } from './transports';
