From 41c50aeed3c85543888aa513b37f207d5797fe9c Mon Sep 17 00:00:00 2001
From: Workers DevProd <workers-devprod@cloudflare.com>
Date: Mon, 2 Oct 2023 18:24:46 +0100
Subject: [PATCH 05/15] Remove unsupported network UI

---
 front_end/panels/network/NetworkPanel.ts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/front_end/panels/network/NetworkPanel.ts b/front_end/panels/network/NetworkPanel.ts
index 71c40a6873..cf0f64f138 100644
--- a/front_end/panels/network/NetworkPanel.ts
+++ b/front_end/panels/network/NetworkPanel.ts
@@ -158,10 +158,6 @@ const UIStrings = {
    *@description Text in Network Panel that is displayed when frames are being fetched.
    */
   fetchingFrames: 'Fetching frames...',
-  /**
-   * @description Text of a button in the Network panel's toolbar that open Network Conditions panel in the drawer.
-   */
-  moreNetworkConditions: 'More network conditions…',
 };
 const str_ = i18n.i18n.registerUIStrings('panels/network/NetworkPanel.ts', UIStrings);
 const i18nString = i18n.i18n.getLocalizedString.bind(undefined, str_);
-- 
2.39.5 (Apple Git-154)

