From b8ca9e6b66ffa2ed4996e6f5519bc09d508c2170 Mon Sep 17 00:00:00 2001
From: Workers DevProd <workers-devprod@cloudflare.com>
Date: Wed, 4 Oct 2023 01:11:14 +0100
Subject: [PATCH 14/15] Show Profiling panel

---
 front_end/entrypoints/main/MainImpl.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/front_end/entrypoints/main/MainImpl.ts b/front_end/entrypoints/main/MainImpl.ts
index a0d739121b..b61092b815 100644
--- a/front_end/entrypoints/main/MainImpl.ts
+++ b/front_end/entrypoints/main/MainImpl.ts
@@ -424,6 +424,7 @@ export class MainImpl {
       ...(Root.Runtime.Runtime.queryParam('isChromeForTesting') ? ['protocol-monitor'] : []),
       Root.Runtime.ExperimentName.AUTHORED_DEPLOYED_GROUPING,
       Root.Runtime.ExperimentName.JUST_MY_CODE,
+      Root.Runtime.ExperimentName.JS_PROFILER_TEMP_ENABLE,
     ]);
 
     Root.Runtime.experiments.cleanUpStaleExperiments();
-- 
2.39.5 (Apple Git-154)

