From 8653c4c2b3a253ad191d74a614b02402618bf735 Mon Sep 17 00:00:00 2001
From: Workers DevProd <workers-devprod@cloudflare.com>
Date: Wed, 4 Oct 2023 01:11:14 +0100
Subject: [PATCH 14/15] Show Profiling panel

---
 front_end/entrypoints/main/MainImpl.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/front_end/entrypoints/main/MainImpl.ts b/front_end/entrypoints/main/MainImpl.ts
index 174cda16c3..c5c3a9f9ae 100644
--- a/front_end/entrypoints/main/MainImpl.ts
+++ b/front_end/entrypoints/main/MainImpl.ts
@@ -436,6 +436,7 @@ export class MainImpl {
       Root.Runtime.ExperimentName.OUTERMOST_TARGET_SELECTOR,
       Root.Runtime.ExperimentName.AUTHORED_DEPLOYED_GROUPING,
       Root.Runtime.ExperimentName.JUST_MY_CODE,
+      Root.Runtime.ExperimentName.JS_PROFILER_TEMP_ENABLE,
     ]);
 
     Root.Runtime.experiments.setNonConfigurableExperiments([
-- 
2.39.5 (Apple Git-154)

