From 6c409326aafb7a6ad7a755f642468e15f05dd4f2 Mon Sep 17 00:00:00 2001
From: Jacob M-G Evans <jacobmgevans@gmail.com>
Date: Fri, 31 Mar 2023 17:11:08 -0500
Subject: [PATCH 02/11] Remove Themes from Setting SubMenu

---
 .../themes/browser/themes.contribution.ts     | 76 +++++++++----------
 1 file changed, 38 insertions(+), 38 deletions(-)

diff --git a/src/vs/workbench/contrib/themes/browser/themes.contribution.ts b/src/vs/workbench/contrib/themes/browser/themes.contribution.ts
index c94be39..22dfc43 100644
--- a/src/vs/workbench/contrib/themes/browser/themes.contribution.ts
+++ b/src/vs/workbench/contrib/themes/browser/themes.contribution.ts
@@ -534,7 +534,7 @@ interface ThemeItem extends IQuickPickItem {
 }
 
 function isItem(i: QuickPickInput<ThemeItem>): i is ThemeItem {
-	return (<any>i)['type'] !== 'separator';
+	return (<any> i)['type'] !== 'separator';
 }
 
 function toEntry(theme: IWorkbenchTheme): ThemeItem {
@@ -655,41 +655,41 @@ registerAction2(class extends Action2 {
 	}
 });
 
-const ThemesSubMenu = new MenuId('ThemesSubMenu');
-MenuRegistry.appendMenuItem(MenuId.GlobalActivity, <ISubmenuItem>{
-	title: localize('themes', "Themes"),
-	submenu: ThemesSubMenu,
-	group: '2_configuration',
-	order: 6
-});
-MenuRegistry.appendMenuItem(MenuId.MenubarPreferencesMenu, <ISubmenuItem>{
-	title: localize({ key: 'miSelectTheme', comment: ['&& denotes a mnemonic'] }, "&&Theme"),
-	submenu: ThemesSubMenu,
-	group: '2_configuration',
-	order: 6
-});
-
-MenuRegistry.appendMenuItem(ThemesSubMenu, {
-	command: {
-		id: SelectColorThemeCommandId,
-		title: localize('selectTheme.label', "Color Theme")
-	},
-	order: 1
-});
-
-MenuRegistry.appendMenuItem(ThemesSubMenu, {
-	command: {
-		id: SelectFileIconThemeCommandId,
-		title: localize('themes.selectIconTheme.label', "File Icon Theme")
-	},
-	order: 2
-});
-
-MenuRegistry.appendMenuItem(ThemesSubMenu, {
-	command: {
-		id: SelectProductIconThemeCommandId,
-		title: localize('themes.selectProductIconTheme.label', "Product Icon Theme")
-	},
-	order: 3
-});
+// const ThemesSubMenu = new MenuId('ThemesSubMenu');
+// MenuRegistry.appendMenuItem(MenuId.GlobalActivity, <ISubmenuItem>{
+// 	title: localize('themes', "Themes"),
+// 	submenu: ThemesSubMenu,
+// 	group: '2_configuration',
+// 	order: 6
+// });
+// MenuRegistry.appendMenuItem(MenuId.MenubarPreferencesMenu, <ISubmenuItem>{
+// 	title: localize({ key: 'miSelectTheme', comment: ['&& denotes a mnemonic'] }, "&&Theme"),
+// 	submenu: ThemesSubMenu,
+// 	group: '2_configuration',
+// 	order: 6
+// });
+
+// MenuRegistry.appendMenuItem(ThemesSubMenu, {
+// 	command: {
+// 		id: SelectColorThemeCommandId,
+// 		title: localize('selectTheme.label', "Color Theme")
+// 	},
+// 	order: 1
+// });
+
+// MenuRegistry.appendMenuItem(ThemesSubMenu, {
+// 	command: {
+// 		id: SelectFileIconThemeCommandId,
+// 		title: localize('themes.selectIconTheme.label', "File Icon Theme")
+// 	},
+// 	order: 2
+// });
+
+// MenuRegistry.appendMenuItem(ThemesSubMenu, {
+// 	command: {
+// 		id: SelectProductIconThemeCommandId,
+// 		title: localize('themes.selectProductIconTheme.label', "Product Icon Theme")
+// 	},
+// 	order: 3
+// });
 
-- 
2.39.2 (Apple Git-143)

