From a0f0970d77717bb6715de290e08720f50693a39c Mon Sep 17 00:00:00 2001
From: Jacob M-G Evans <jacobmgevans@gmail.com>
Date: Fri, 31 Mar 2023 17:11:28 -0500
Subject: [PATCH 04/11] Remove Profile from Setting SubMenu

---
 .../browser/userDataProfile.ts                | 46 +++++++++----------
 1 file changed, 23 insertions(+), 23 deletions(-)

diff --git a/src/vs/workbench/contrib/userDataProfile/browser/userDataProfile.ts b/src/vs/workbench/contrib/userDataProfile/browser/userDataProfile.ts
index 5312130..b95cbb7 100644
--- a/src/vs/workbench/contrib/userDataProfile/browser/userDataProfile.ts
+++ b/src/vs/workbench/contrib/userDataProfile/browser/userDataProfile.ts
@@ -82,7 +82,7 @@ export class UserDataProfilesWorkbenchContribution extends Disposable implements
 	}
 
 	private registerActions(): void {
-		this.registerProfileSubMenu();
+		// this.registerProfileSubMenu();
 		this._register(this.registerSwitchProfileAction());
 
 		this.registerProfilesActions();
@@ -97,28 +97,28 @@ export class UserDataProfilesWorkbenchContribution extends Disposable implements
 		this.registerDeleteProfileAction();
 	}
 
-	private registerProfileSubMenu(): void {
-		const getProfilesTitle = () => {
-			return localize('profiles', "Profiles ({0})", this.userDataProfileService.currentProfile.name);
-		};
-		MenuRegistry.appendMenuItem(MenuId.GlobalActivity, <ISubmenuItem>{
-			get title() {
-				return getProfilesTitle();
-			},
-			submenu: ProfilesMenu,
-			group: '1_profiles',
-			order: 1,
-		});
-		MenuRegistry.appendMenuItem(MenuId.MenubarPreferencesMenu, <ISubmenuItem>{
-			get title() {
-				return getProfilesTitle();
-			},
-			submenu: ProfilesMenu,
-			group: '1_profiles',
-			order: 1,
-			when: PROFILES_ENABLEMENT_CONTEXT,
-		});
-	}
+	// private registerProfileSubMenu(): void {
+	// 	const getProfilesTitle = () => {
+	// 		return localize('profiles', "Profiles ({0})", this.userDataProfileService.currentProfile.name);
+	// 	};
+	// 	MenuRegistry.appendMenuItem(MenuId.GlobalActivity, <ISubmenuItem> {
+	// 		get title() {
+	// 			return getProfilesTitle();
+	// 		},
+	// 		submenu: ProfilesMenu,
+	// 		group: '1_profiles',
+	// 		order: 1,
+	// 	});
+	// 	MenuRegistry.appendMenuItem(MenuId.MenubarPreferencesMenu, <ISubmenuItem> {
+	// 		get title() {
+	// 			return getProfilesTitle();
+	// 		},
+	// 		submenu: ProfilesMenu,
+	// 		group: '1_profiles',
+	// 		order: 1,
+	// 		when: PROFILES_ENABLEMENT_CONTEXT,
+	// 	});
+	// }
 
 	private readonly profilesDisposable = this._register(new MutableDisposable<DisposableStore>());
 	private registerProfilesActions(): void {
-- 
2.39.2 (Apple Git-143)

