From dca4f37b1b0c5b6488896e58b1fb649502b4b2dd Mon Sep 17 00:00:00 2001
From: Workers DevProd <workers-devprod@cloudflare.com>
Date: Fri, 25 Aug 2023 16:48:29 -0400
Subject: [PATCH 13/13] Hide File Explorer by default

---
 src/vs/workbench/browser/layout.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/vs/workbench/browser/layout.ts b/src/vs/workbench/browser/layout.ts
index b522eaa..64933ff 100644
--- a/src/vs/workbench/browser/layout.ts
+++ b/src/vs/workbench/browser/layout.ts
@@ -534,6 +534,7 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 			initialization: initialLayoutState,
 			runtime: layoutRuntimeState,
 		};
+	  this.stateModel.setRuntimeValue(LayoutStateKeys.SIDEBAR_HIDDEN, true);
 
 		// Sidebar View Container To Restore
 		if (this.isVisible(Parts.SIDEBAR_PART)) {
-- 
2.39.2 (Apple Git-143)

